bust-a-move-framework/src/main/java/bustamove/gamestate/StartScreen.java

```diff
@@ -47,6 +53,8 @@ public final int getID() {
     public final void init(final GameContainer game,
                            final StateBasedGame stateBasedGame)
             throws SlickException {
+        Button.setFont(font);
```

Comment:
You are setting the font of ALL the buttons and text elements in the StartScreen class? In my opinion it would be better if it was in a more intuitive position for if we ever want to change the font, like gameconfig?